### PR TITLE
Companion for substrate/pull/8445

### PR DESCRIPTION
**Pull request by kianenigma**
Companion for https://github.com/paritytech/substrate/pull/8445

@rrtti please apply all other changes that we need to the society.

### CODE REVIEW

File: runtime/kusama/src/lib.rs [hunk 1]
```diff
@@ -791,6 +791,7 @@ parameter_types! {
 	pub const PeriodSpend: Balance = 500 * DOLLARS;
 	pub const MaxLockDuration: BlockNumber = 36 * 30 * DAYS;
 	pub const ChallengePeriod: BlockNumber = 7 * DAYS;
+	pub const MaxCandidateIntake: u32 = 1;
```
Comment (shawntabrizi) [inline runtime/kusama/src/lib.rs:794]: With intake of only 1, likely the only people to get into society will be those that bid 0.

just fyi

Comment (kianenigma) [reply runtime/kusama/src/lib.rs:794]: Society is full currently, and @rrtti wanted to make a few changes, including the above. I presume he has good reasons for it.

Comment (shawntabrizi) [reply runtime/kusama/src/lib.rs:794]: yeah, understandable, but in terms of what this change will do in the long term, that people with bid amounts probably don't get in anymore.

**Review by shawntabrizi — APPROVED** [review]

**Comment by kianenigma** [comment]
whyy did the bot fail here again?

**Comment by kianenigma** [comment]
thanks @shawntabrizi

**Comment by kianenigma** [comment]
bot merge

**Pull request merged**